Vagrantfile

```diff
@@ -4,23 +4,20 @@
 
 Vagrant.configure(2) do |config|
 
-    if ENV['VAGRANT_BOX']
-      config.vm.box = ENV['VAGRANT_BOX']
-    else
-      config.vm.box = "centosah/7alpha"
-      config.vm.box_url = 'https://ci.centos.org/artifacts/sig-atomic/centos-continuous/images-alpha/cloud/latest/images/centos-atomic-host-7-vagrant-libvirt.box'
-    end
+    config.vm.box = "centosah/7alpha"
+    config.vm.box_url = 'https://ci.centos.org/artifacts/sig-atomic/centos-continuous/images-alpha/cloud/latest/images/centos-atomic-host-7-vagrant-libvirt.box'
 
     config.vm.hostname = "centosah-dev"
 
     config.vm.define "vmcheck" do |vmcheck|
     end
 
     if Vagrant.has_plugin?('vagrant-sshfs')
-      config.vm.synced_folder ".", "/var/home/vagrant/sync", type: 'sshfs'
+      config.vm.synced_folder ".", "/var/roothome/sync", type: 'sshfs'
+      File.write('.vagrant_sshfs', '')
```

Comment:
Hum, maybe `.vagrant/sshfs-in-use` or something, to avoid polluting the toplevel?